package.json

```diff
@@ -4,9 +4,15 @@
   "description": "A native node module for accessing the windows registry",
   "main": "dist/index.js",
   "typings": "dist/index.d.ts",
+  "files": [
+	"dist/index.js",
+	"dist/index.d.ts",
+	"dist/winregistry.node"
+  ],
   "scripts": {
-    "compile": "tsc -p ./",
-    "test": "mocha -r ts-node/register test/**/*.test.ts"
+	"prepublishOnly": "npm run compile && npm run test",
+	"compile": "[ -f binding.gyp ] && tsc -p ./ && node-gyp rebuild && cp build/Release/winregistry.node dist/",
```

Comment:
We currently compile the package downstream on different architectures. I'd rather not bring in this change because it'd require adjusting CI to build for different architectures, unless there's a good reason to do that step here rather than downstream.